server/ServerApp.cpp

```diff
@@ -1380,7 +1380,7 @@ void ServerApp::GenerateUniverse(std::map<int, PlayerSetupData>& player_setup_da
             seed = static_cast<unsigned int>(h);
         } catch (...) {}
     }
-    if (GetGalaxySetupData().m_seed.empty()) {
+    if (GetGalaxySetupData().GetSeed().empty() || GetGalaxySetupData().GetSeed() == "RANDOM") {
```

Comment:
What happens if you click the button to randomly generate a seed, then put the cursor in the text box to type a seed and press enter. Presumably it will then set the seed to whatever the localized version of "Random" is, and not actually generate a random seed.

Reply:
Good point, I've changed to disable the edit control when set to random, and convert an empty seed value to the random toggle.

I mildly prefer the dice graphic so left it unchanged, would you prefer a checkbox or an alternate rendering when toggled?

Reply:
Different graphic is OK as long as it's clear when it's enabled or disabled... (eg. not like the tech status display toggles)

Reply:
Changing to checkbox does not look appropriate and not sure what the usability issue is with tech controls, looks best to leave it for now.

Reply:
The filters for techs look almost or exactly the same when on or off. Only when moused over do they look different.